deploy/dracon/chart/values.yaml

```diff
@@ -64,7 +64,14 @@ enrichmentDB:
   # enrichment database
   connectionStr: ""
 
+dracon-components:
```

Comment:
I don't see any change in the Helm manifests where this is used. why is it here?